microformats.gemspec

```diff
@@ -16,14 +16,14 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/indieweb/microformats-ruby'
   spec.license       = 'CC0-1.0'
 
-  spec.files         = `git ls-files`.split("\x0").select { |f| f.match(%r{^(bin|lib)/}) }
-
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(spec|vendor)/}) }
   spec.bindir        = 'bin'
-  spec.executables   = ['microformats']
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
```

Comment:
Future-proofing the addition of more executables. 👍

Reply:
I disagree. I think executables should be explicit. Like a ship's manifest. Also, a new executable is added so rarely (hasn't even happen once yet!) that it's not much an effort to add it to this list if it does happen.

Reply:
Point taken. I'll revert this change. 👍

Reply:
👍🏻